jsconfig-example.json

```diff
@@ -36,92 +36,104 @@
     "site",
 
     // Necessary action:
-    // Uncomment paths that are NOT your application(s) to improve IntelliSense performance.
-    // The more paths you exclude, the faster and more reliable IntelliSense
-    // will be for code suggestions and navigating to files from imports.
-    // Note: Changes here only affect your local development environment.
+    // You must comment out or remove your specific application to get intellisense
+    // (auto-complete, hover-hints, and go-to-definition behavior).
+    // If your application is NOT listed here, then you WILL get intellisense.
+    // The reason we exclude everything by default is because there is a
+    // limit on the number of files the IDE can index for intellisense.
```

Comment:
Hopefully these instructions are more clear now. Several developers misinterpreted it before. Also deciding to exclude everything by default now, so the user just has to make 1 change, instead of uncomment everything except their own.

From my experience, it doesn't matter whether you default to all included or all excluded, because if you include everything, intellisense doesn't work anyways, because there's too much to load.